sdk-lib/src/androidTest/java/cash/z/ecc/android/sdk/integration/service/ChangeServiceTest.kt

```diff
@@ -79,99 +71,8 @@ class ChangeServiceTest : ScopedTest() {
             twig(it)
         }.getOrElse { return }
 
-        assertTrue(result > network.saplingActivationHeight)
-    }
-
-    @Test
-    fun testCleanSwitch() = runBlocking {
```

Comment:
are you planning to create these later on ? otherwise the name of the test class should be changed

Reply:
We won't support switching right now.  Instead, SDK consumers should stop the synchronizer and start it again with new arguments.

In the future, the networking abstraction may take care of automatically switching between multiple endpoints if they are available.